### ISSUE TITLE
Add a new user type: "editor user" 

### ISSUE THREAD

**Issue by yu-iskw** [polyaxon/polyaxon#315]
### Feature motivation
At the moment, we have two user type which are superuser and normal user. Normal users have only the permission to read project of other users. If we want to grant writer permissions of other users to normal users, we must to give superuser role to them right now. But, in my opinion, adding such a strong permission just to give write permission to users is too much. In my opinion, it would be great to add the middle user type between superuser and normal user. What do you think of my idea?

### Feature description
For instance, Google Cloud offers 4 major user types Browser, Viewer, Editor and Owner. The types from view to owner can inspire us. It mean, I suggest that we will add "editor user" who can writer projects of other members, but who can not have permission to grant and so on.

**Comment by mouradmourafiq** [comment]
sorry for late answer on this, I had already answered this question several times on the slack channel in the past, I was waiting until the new site is up.

The short answer is that: I made roles for some companies a couple of months ago, and I abstracted this feature in the EE version also a couple of months ago, [see](https://docs.polyaxon.com/concepts/features/#polyaxon-ee).

Many stuff get moved from EE to CE, so this also could make it at some point to the CE version.

So I will close this for now.
